Angular/AsignacionAngular/RunForestRun/src/app/main-content/main-content.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { AgentServiceService} from './agent-service.service';
+import { Observable } from 'rxjs';
+
+
+@Component({
+  selector: 'app-main-content',
+  templateUrl: './main-content.component.html',
+  styleUrls: ['./main-content.component.scss']
+})
+export class MainContentComponent implements OnInit {
+    agents$: Object;
```

Comment:
Avoid using 'Object' type
In angular exist the models, that are kind of dev defined types
For example, you can create an Agent model, and you defined which properties it has, like name, cost per hour, etc and here you would declare an array of agents